AK/SourceGenerator.h

```diff
@@ -37,7 +37,15 @@ class SourceGenerator {
 
     SourceGenerator fork() { return SourceGenerator { m_builder, m_mapping, m_opening, m_closing }; }
 
-    void set(StringView key, String value) { m_mapping.set(key, move(value)); }
+    void set(StringView key, String value)
+    {
+        if (!key.is_empty() && key[0] == '@') {
```

Comment:
```suggestion
        if (!key.starts_with('@')) {
```

Reply:
Thanks. Actually, I'm going to check if it *contains* `@`, since that's always invalid.